primitives/core/src/generic_discord_role.rs

```diff
@@ -17,9 +17,10 @@
 use codec::{Decode, Encode, MaxEncodedLen};
 use scale_info::TypeInfo;
 
+use crate::{ContestType, SoraQuizType};
+
 #[derive(Encode, Decode, Clone, Debug, PartialEq, Eq, MaxEncodedLen, TypeInfo)]
 pub enum GenericDiscordRoleType {
```

Comment:
Maybe this can be flatten (remove this enum completely) ?I think I know the intentions but it looks like we are trying to group assertions by data providers which may be a subject of change (sorry for late notice).

Reply:
Maybe we refactor this once the generic discord VC design is aligned in seperate PR?